src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
```

Comment:
```suggestion
  user: User;
```
this component doesn't make sense if `user === null`, so it's better to check it in `TodoInfo` component and depending on the result render or not `UserInfo` component (use [conditional rendering](https://react.dev/learn/conditional-rendering))